compiler/noirc_frontend/src/ast/mod.rs

```diff
@@ -359,6 +359,18 @@ pub enum ItemVisibility {
     Public,
     Private,
     PublicCrate,
+    PublicSuper,
+}
+
+impl std::fmt::Display for ItemVisibility {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            Self::Public => write!(f, "pub"),
+            Self::Private => write!(f, "priv"),
```

Comment:
Shouldn't `priv` print nothing? That way the formatter can use this function as well